examples/components/single_node/pandas/README.MD

```diff
@@ -5,8 +5,7 @@
 Run the following command to upload a Pandas dataframe to a Google Cloud Storage bucket:
 
 ```
-python3 data_loading_pandas.py --extra-args '{"project_id": "storied-landing-366912"}' \
- --output-manifest <local_path> \
-  --metadata-args '{"run_id":"test","component_name":"test_component", \
-  "artifact_bucket":"storied-landing-366912-kfp-output"}'
+--args '{"project_id": "soy-audio-379412", "run_id":"test","component_name":"test_component", \
```

Comment:
```suggestion
--args '{"project_id": "soy-audio-379412", "run_id":"test", "component_name":"test_component", \
```

Reply:
Isn't it a bit weird that we need to pass the name of the component to the component itself?

Reply:
Run_id and project_id makes sense as metadata as those are different each time you run a component, but the name of the component seems inherent to the component itself